### PR TITLE
Release/18.04

### PR DESCRIPTION
**Pull request by steverydz**
This is the 18.04 release changes. This PR is so we can have a working demo of the completed work.

### CODE REVIEW

File: templates/download/server/s390x.html [hunk 1]
```diff
@@ -18,8 +18,9 @@ <h1>Ubuntu Server for IBM Z and LinuxONE</h1>
       <h2>Ubuntu Server {{lts_release_full}}</h2>
       <div class="u-equal-height p-divider">
         <div class="col-6 p-divider__block">
-          <p class="p-card__content">IBM Z and LinuxONE leverage open technology solutions to meet the demands of the new application economy. They combine the best of open source to offer IBM&rsquo;s enterprise customers unprecedented performance, flexibility, security and resiliency.</p>
+          <p class="p-card__content">IBM Z and LinuxONE systems leverage open technology solutions to meet the demands of the new application economy. They combine the best of open source to offer IBM&rsquo;s enterprise customers unprecedented performance, flexibility, security and resiliency. Ubuntu {{lts_release_full was released for general availability (GA) in April and can now be downloaded by all IBM Z and LinuxONE system customers.</p>
```
Comment (anthonydillon) [inline templates/download/server/s390x.html]: `{{lts_release_full` is missing its closing `}}`

File: templates/download/server/thank-you-s390x.html [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+{% endblock %}
+
+{% block content %}
+
+<div class="p-strip--light is-deep is-bordered">
+  <div class="row">
+    <div class="col-12">
+      <h1>Thanks for downloading Ubuntu for LinuxONE and z&nbsp;Systems</h1>
+      <p>Thank you for downloading Ubuntu {{lts_release_full_with_point}} for IBM Z and LinuxONE. Your ISO download should start automatically.</p>
+      <p>If it doesn&rsquo;t, or for alternative options <a href="http://cdimage.ubuntu.com/releases/18.04/release">visit the download page</a>.</p>
```
Comment (anthonydillon) [inline templates/download/server/thank-you-s390x.html]: external link

File: templates/server/index.html [hunk 3]
```diff
@@ -372,7 +374,7 @@ <h3 class="p-muted-heading u-align--center">All guests are welcome on Ubuntu</h3
         <h2 id="a-thriving-community">A thriving community</h2>
         <p>Exchange expertise and ideas with thousands of other IT professionals</p>
         <p>Want to talk to other Ubuntu users straightaway? Share ideas and get advice and help from our large, active community of IT professionals. As a community, we set high standards for friendliness and tolerance, we welcome your questions and contributions!</p>
-        <p><a class="p-link--external" href="http://community.ubuntu.com">Join the community</a></p>
+        <p><a class="p-link--external" href="/community">Join the community</a></p>
```
Comment (anthonydillon) [inline templates/server/index.html]: This is not an external link.

File: templates/takeovers/_1804-takeover.html [hunk 4]
```diff
@@ -0,0 +1,12 @@
+<section class="p-strip p-takeover--1804-release">
+  <div class="row">
+    <div class="p-takeover__body u-clearfix">
+      <div class="col-8">
+        <h1 class="p-takeover__title">Ubuntu 18.04 LTS out now</h1>
+        <p class="p-heading--four">Optimised for multi-cloud infrastructure, machine learning, AI and software development</p>
+        <p class="p-takeover__image u-hidden--medium u-hidden--large"><img src="{{ ASSET_SERVER_URL }}99cee55b-beaver.svg" alt=""></p>
```
Comment (anthonydillon) [inline templates/takeovers/_1804-takeover.html]: It's `u-hide--medium` and `u-hide--large`

**Comment by steverydz** [comment]
@anthonydillon I've done your changes here: https://github.com/canonical-websites/www.ubuntu.com/pull/3057

It's too complicated to try and push to this branch as it's an upstream.

**Comment by frankiegranato** [comment]
UX will be ongoing after merging.

**Pull request merged**